glean-core/src/lib.rs

```diff
@@ -38,7 +38,6 @@ pub mod ping;
 pub mod storage;
 mod util;
 
-#[cfg(feature = "upload")]
```

Comment:
If you kept this and tried to guard the field below?

Reply:
If I keep this, whener I try to use the upload stuff below would produce an error, and each of the erroring line would require this annotation.

Reply:
Yes, I'd be fine with this unless there's a better option. We don't really want this enabled on master untill we test the full thing.